src/components/game/graphics/sections/Palette.js

```diff
@@ -0,0 +1,39 @@
+import { drawBackground, STAGE_CELL_SIZE, PALETTE_X0, PALETTE_Y0, PALETTE_WIDTH, PALETTE_HEIGHT } from "../Layout";
+import { drawPiece, updatePiece } from "../objects/Piece";
+
+// See comment in ./Board.js for more about convention with objects and sections
+export function renderPalette(canvas, palette) {
```

Comment:
I would add comments that need to be addressed but this whole thing will get overwritten in another issue when the Palette feature is actually added